nomenclature/cli.py

```diff
@@ -93,3 +96,68 @@ def cli_valid_project(
     """
     assert_valid_yaml(path)
     assert_valid_structure(path, definitions, mappings, required_data, dimensions)
+
+
+@cli.command("run-region-processing")
```

Comment:
Maybe rename to `check-region-aggregation` for consistency with the function?

Reply:
Good point